features/guidance/rotary.feature

```diff
@@ -90,19 +90,19 @@ Feature: Rotary
             | bcegb  | roundabout |
 
        When I route I should get
-           | waypoints | route       | turns                      |
-           | b,d       | bcegb,cd,cd | depart,bcegb-exit-1,arrive |
-           | b,f       | bcegb,ef,ef | depart,bcegb-exit-2,arrive |
-           | b,h       | bcegb,gh,gh | depart,bcegb-exit-3,arrive |
-           | c,f       | bcegb,ef,ef | depart,bcegb-exit-1,arrive |
-           | c,h       | bcegb,gh,gh | depart,bcegb-exit-2,arrive |
-           | c,a       | bcegb,ab,ab | depart,bcegb-exit-3,arrive |
-           | e,h       | bcegb,gh,gh | depart,bcegb-exit-1,arrive |
-           | e,a       | bcegb,ab,ab | depart,bcegb-exit-2,arrive |
-           | e,d       | bcegb,cd,cd | depart,bcegb-exit-3,arrive |
-           | g,a       | bcegb,ab,ab | depart,bcegb-exit-1,arrive |
-           | g,d       | bcegb,cd,cd | depart,bcegb-exit-2,arrive |
-           | g,f       | bcegb,ef,ef | depart,bcegb-exit-3,arrive |
+           | waypoints | route       | turns                           |
+           | b,d       | bcegb,cd,cd | depart,exit rotary right,arrive |
+           | b,f       | bcegb,ef,ef | depart,exit rotary right,arrive |
+           | b,h       | bcegb,gh,gh | depart,exit rotary right,arrive |
```

Comment:
@oxidase again, the exit number is part of the instruction. We can add it to cucumber as well. I would be careful, though, since the exit-number would change based on you driving in the roundabout.
So you may already have passed an exit before the route is in, you may have passed a second one before it is announced. So in case of re-routing on a roundabout, I would expect that `exit now` would actually be more helpful than take the 3rd exit, which by now might be the first.